### PR TITLE
EKCO -- Kotsadm addon HA management

### PR DESCRIPTION
**Pull request by sgalsaleh**
[SC-61372](https://app.shortcut.com/replicated/story/61372/have-ekco-manage-the-scale-up-of-rqlite-when-3-nodes-is-reached)

### CODE REVIEW

File: src/markdown-pages/add-ons/ekco.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ spec:
     shouldDisableRestartFailedEnvoyPods: false
     envoyPodsNotReadyDuration: 5m
     minioShouldDisableManagement: false
+    shouldDisableKotsadmManagement: false
 ```
 
 flags-table
```
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply src/markdown-pages/add-ons/ekco.md:36]: It is though?

File: static/versionDetails.json [hunk 2]
```diff
@@ -360,7 +360,12 @@
       "flag": "disableS3",
       "description": "Migrate application archives and support bundles from S3 and use a local volume in the kotsadm statefulset instead. Also changes nfs and hostpath snapshots from using minio to using the local-volume-provider plugin. The migration process is irreversible and will replace the kotsadm deployment with a statefulset. While this reduces KOTS's dependency on an object store for the application archives and support bundles, an object store is still required to deploy KOTS. Defaults to 'false.'",
       "type": "boolean"
-    }
+    },
+		{
```
Comment (emosbaugh) [inline static/versionDetails.json:364]: indentation is off

**Review by emosbaugh — APPROVED** [review]

File: src/markdown-pages/add-ons/ekco.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ajp-io) [reply src/markdown-pages/add-ons/ekco.md:36]: It's in the sample spec at the top. But right below that is a table, and I don't see it in there.

File: static/versionDetails.json [hunk 3]
```diff
@@ -360,6 +360,11 @@
       "flag": "disableS3",
       "description": "Migrate application archives and support bundles from S3 and use a local volume in the kotsadm statefulset instead. Also changes nfs and hostpath snapshots from using minio to using the local-volume-provider plugin. The migration process is irreversible and will replace the kotsadm deployment with a statefulset. While this reduces KOTS's dependency on an object store for the application archives and support bundles, an object store is still required to deploy KOTS. Defaults to 'false.'",
       "type": "boolean"
+    },
+    {
+      "flag": "kotsadmShouldDisableManagement",
+      "description": "Prevent high availability (HA) Kotsadm from being enabled and do not manage existing HA Kotsadm components",
```
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply static/versionDetails.json]: Hmm, sure, what do you propose?

**Review by ajp-io — APPROVED** [review]

File: static/versionDetails.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ajp-io) [reply static/versionDetails.json]: I approved and forgot we had this thread open. Going into an interview, but I'll comment shortly.

[the remark this answers is not in the document]
Comment (sgalsaleh) [reply static/versionDetails.json]: that's kind of the reason i added "components" at the end

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**